support-frontend/app/services/HttpIdentityService.scala

```diff
@@ -105,32 +105,17 @@ class IdentityService(apiUrl: String, apiClientToken: String)(implicit wsClient:
       .subflatMap(resp => resp.json.validate[CreateSignInTokenResponse].asEither.leftMap(_.mkString(",")))
   }
 
-  private def getHeaders(request: RequestHeader): List[(String, String)] = List(
-    "X-GU-ID-Client-Access-Token" -> Some(s"Bearer $apiClientToken"),
-    "X-GU-ID-FOWARDED-SC-GU-U" -> request.cookies.get("SC_GU_U").map(_.value),
```

Comment:
now we only call getUserIdFromEmail, so the user would not be logged in anyway (guest mode), so forwarding the SC_GU_U could only cause problems, not benefits